packages/hash/api/src/graphql/resolvers/ontology/model-mapping.ts

```diff
@@ -63,10 +65,25 @@ export const mapSubgraphToGql = (subgraph: Subgraph): SubgraphGql => {
           // These types are compatible with the Type System package's types
           case "dataType":
           case "linkType":
-          case "entity":
           case "link": {
             return [identifier, vertex];
           }
+          case "entity": {
+            const persistedEntity = vertex.inner;
+            const entityVertex: EntityVertex = {
+              kind: vertex.kind,
+              inner: {
+                accountId: persistedEntity.metadata.identifier.ownedById,
+                entityId: persistedEntity.metadata.identifier.entityId,
+                entityTypeId: persistedEntity.metadata.entityTypeId,
+                entityVersion: persistedEntity.metadata.identifier.version,
+                linkedEntities: [],
```

Comment:
Assuming this needs to be removed once the sub-graph contains links, can we add a `@todo` and an asana task link so we don't forget?
```suggestion
                linkedEntities: [],
```

Reply:
I'm not sure this is even the right place to add a `@todo` if I'm honest. There are so many places we'd need to update I'm not sure it's even worth our time trying to capture them rather than just doing it.

If we delete them from the typedefs, and there is a task to do soon to rework the entity GraphQL types, then the type errors will take us to all of these places (I hope).

Reply:
Why not remove this as you removed `entityType`? I don't think `linkedEntities` from this type definition is being used anywhere either

Reply:
Hm, it didn't _seem_ to cause any compilation failures so I have removed it as you suggest. My worry about deleting these things is that we aren't spending time to properly find the other places to delete. We have a **load** of files with `"linkedEntities"` in them, including files entirely dedicated to them. I think it's going to become increasingly difficult to track what is actually redundant if we keep doing half measures.

Reply:
https://github.com/hashintel/hash/pull/1242/commits/a2764cf3bae5707c823f9508780a7358d7883fe6

Reply:
> We have a load of files with "linkedEntities" in them, including files entirely dedicated to them

True, ideally we will be able to remove these once we have links in the subgraphs